README.md

```diff
@@ -5,6 +5,7 @@
 Node 4.x (and above) is **required**. We suggest using [nvm](https://github.com/creationix/nvm) or [n](https://github.com/tj/n) to keep multiple Node versions on your system.
 
 
+
```

Comment:
can you write unit tests, integration tests, docs, and create a wiki, and send out an email to the team for this change?